app/(dashboard)/settings/sectionWrapper.tsx

```
@@ -12,11 +12,12 @@ type SectionWrapperProps = {
   className?: string;
   action?: React.ReactNode;
   children: React.ReactNode;
+  "data-testid"?: string;
 };
 
-const SectionWrapper = ({ title, description, fullWidth, className, action, children }: SectionWrapperProps) => {
+const SectionWrapper = ({ title, description, fullWidth, className, action, children, "data-testid": dataTestId }: SectionWrapperProps) => {
```

ESLint

[error] Replace `·title,·description,·fullWidth,·className,·action,·children,·"data-testid":·dataTestId·` with `⏎··title,⏎··description,⏎··fullWidth,⏎··className,⏎··action,⏎··children,⏎··"data-testid":·dataTestId,⏎`

(prettier/prettier)